app/components/ui/molecules/DashboardListItem.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
+import styled, { withTheme } from 'styled-components'
+import { Flex, Box } from 'grid-styled'
+import { differenceInCalendarDays, format } from 'date-fns'
+import { th } from '@pubsweet/ui-toolkit'
+import PropTypes from 'prop-types'
+
+const dashboardStatusColor = (status, theme) =>
+  ({
+    1: theme.colorTextSecondary,
+    2: theme.colorError,
+    3: theme.colorSuccess,
+  }[status])
```

Comment:
Rather than returning the actual colour from this function, you can return the colour name (e.g. `colorError`). Then you can use that in `StatusBox` to index into the theme. That way you don't have to wrap the component in `withTheme`.

Reply:
Sorry, how is theme added to props if I don't wrap it in `withTheme`?